pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 requires-python = ">=3.9"
 dependencies = [
     "antsibull-changelog >= 0.14.0",
-    "antsibull-core >= 2.0.0, < 3.0.0",
+    "antsibull-core >= 2.0.0, <= 3.0.0.dev0",
```

Comment:
Why is this part necessary?

Reply:
Ah right, so https://github.com/ansible-community/antsibull-core/pull/127 can work.